hooks/post-checkout

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+git submodule update
```

Comment:
Is the check nessessary? Seems already coverd in init script.

Reply:
The init script is only run once, when setting up the repo. This runs after switching branches, to ensure that the cumulus version is correct on the new branch.